CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+Create a new branch for your work:
+
+    git checkout -b <my_branch_name>
+
+Code and document your new feature:
+    
+    We appreciate well organized commits, commit messages
+
+Push your changes up to GitHub:
+    
+    git push -u origin <my_branch_name>
```

Comment:
I don't think users outside of Localytics can push new branches to the repo. We should use the fork workflow instead.